rust/src/request.rs

```diff
@@ -83,79 +111,108 @@ impl Proxy {
         }
     }
 
+    pub async fn get_global_seeds(&self) -> Result<Option<SeedDict>, ClientError> {
```

Comment:
the naming `global_seeds` is quite unfortunate. the (local) seeds are used to generate the local masks, but there is no equivalent for the global mask (since the global mask is aggregated from the local masks and no seed is known for the global mask).

this is somewhat connected to the task PB-778. we may name it like `get_all_seeds` or sth similar, but in the end we should get the overall naming right in the near future (as a note to all of us).

Reply:
I'll go with `get_all_seeds` for now, thanks!

Reply:
Actually it's not even being used so I just removed it